GoogleSignIn/Sources/GIDEMMErrorHandler.h

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <TargetConditionals.h>
+
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
```

Comment:
Currently, EMM support is disabled for Mac Catalyst, so we should be able to exclude that case here.  See #77 for details.

We should update the condition used in #77 to only enable EMM support for iOS.

Reply:
Ack